thops25-homework/Tuck-Hauptfuhrer/hw_3/hw3.rb

```diff
@@ -0,0 +1,62 @@
+puts "Find the length of strings in an array:"
+
+def length_finder(input_array)
+  array_length=[]
+  input_array.each do |i| 
+    array_length.push(i.length)
+  end  
+  return array_length
+end
+
+puts "Find the frequency of a string in a sentence:"
+
+def find_frequency(sentence, word)
+  word_downcase = word.downcase
+  sentence_words=[]
+  sentence_words = sentence.split()
+  c = []
+  sentence_words.each {|i| c.push(i.downcase)} 
+  b = c.count(word_downcase)    
+  return b
+end
+
+puts "Select random elements from an array:"
+
+def random_select(array, n)
+  a = []
+  b = array.length - 1
+  i = 0
```

Comment:
@thops25 With the exception of iterators, it is Ruby convention variables be named descriptively. I should be able to infer the responsibility of each variable easily because they are concisely named. No more single letter variables, please. :octopus: